From 12e0fe75aa11529a5c952e29b16373f7282142aa Mon Sep 17 00:00:00 2001
From: Sivashankari Madhavan <quic_sivamadh@quicinc.com>
Date: Thu, 22 Sep 2022 16:33:47 +0530
Subject: [PATCH] ath12k: Fix flush error options 71 to 127 are not present in
 htt_stats

While print the Htt flush errors considering the min_t from tag_len and
HTT_TX_PDEV_MAX_FLUSH_REASON_STATS(71) for printing. Here HTT_TX_PDEV_MAX_FLUSH_REASON_STATS
is less than of tag_len. So only stats printed till 70.

Changed the HTT_TX_PDEV_MAX_FLUSH_REASON_STATS as 150.

Signed-off-by: Sivashankari Madhavan <quic_sivamadh@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/debugfs_htt_stats.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath12k/debugfs_htt_stats.h b/drivers/net/wireless/ath/ath12k/debugfs_htt_stats.h
index 1fb89a2..f0064e6 100644
--- a/drivers/net/wireless/ath/ath12k/debugfs_htt_stats.h
+++ b/drivers/net/wireless/ath/ath12k/debugfs_htt_stats.h
@@ -181,7 +181,7 @@ enum htt_ppdu_stats_ru_size {
 	HTT_PPDU_STATS_RU_996x4,
 };
 
-#define HTT_TX_PDEV_MAX_FLUSH_REASON_STATS     71
+#define HTT_TX_PDEV_MAX_FLUSH_REASON_STATS     150
 #define HTT_TX_PDEV_MAX_SIFS_BURST_STATS       9
 #define HTT_TX_PDEV_MAX_SIFS_BURST_HIST_STATS  10
 #define HTT_TX_PDEV_MAX_PHY_ERR_STATS          18
-- 
2.17.1

